wildfly/commit-markable-resource/pom.xml

```diff
@@ -75,9 +54,10 @@
                 <version>${version.surefire.plugin}</version>
                 <configuration>
                     <skip>false</skip>
-                    <argLine>${surefire.argline}</argLine>
+                    <argLine>${surefire.argline} ${modular.jdk.args}</argLine>
```

Comment:
Very strange you need to use ${modular.jdk.args} twice :confused:

Reply:
Yes, this is strange. Actually now this quickstart does not need the modular.jdk.args (which is empty now) therefore I could remove them. WDYT?

Reply:
Yes please :-)